components/chat/completion.tsx

```diff
@@ -159,24 +200,43 @@ const Completion = ({ messages, onSubmit, paramsObj = {}, clearInitMessage }: Pr
                 }}
               >
                 {paramsOpts.map((item) => (
-                  <Option key={item.key} value={item.value}>
-                    {item.key}
-                  </Option>
+                  <AntdTooltip title={paramsInfoObj?.[item.key]} key={'tp-' + item.key} variant='solid' placement='right'>
+                    <Option key={item.key} value={item.value}>
+                      {item.key}
+                    </Option>
+                  </AntdTooltip>
                 ))}
               </Select>
             </div>
           )}
-          <Input
+          <Textarea
```

Comment:
The conversation input box has been refactored in the next version, so there is no need to update it here.